admin/config.js

```diff
@@ -123,10 +123,10 @@ module.exports = {
   },
   Monitoring: {
     ApplicationInsights: {
-      Key: process.env.APPINSIGHTS_INSTRUMENTATIONKEY,
+      Key: process.env.APPINSIGHTS_INSTRUMENTATIONKEY, // @deprecated
```

Comment:
should we instead remove this if we are wholesale moving away from it, and there are no more usages?

Reply:
I agree - good spot!!